mqtt/mqtt-broker/src/connection/mod.rs

```diff
@@ -169,15 +169,14 @@ where
                 broker_handle.send(message)?;
 
                 let (outgoing, incoming) = codec.split();
+                let (outgoing_processor, incoming_processor) = make_processor.make(&client_id);
 
                 // prepare processing incoming packets
-                let incoming_processor = make_processor.make_incoming(&client_id);
                 let incoming_task =
-                    incoming_task(client_id.clone(), incoming, broker_handle.clone(), incoming_processor);
+                    incoming_task(client_id.clone(), incoming, broker_handle.clone(),incoming_processor);
```

Comment:
fmt?

Reply:
seems like there is a bug in the formatter. all checks have passed with ✅